index.js

```diff
@@ -43,21 +52,20 @@ exports.speak = function(voice, text, callback) {
     childD.stdin.end(pipedData);
   }
 
-
-  childD.stderr.on('data', function(data){ console.log(data); });
-  childD.stdout.on('data', function(data){ console.log(data); });
-
+  childD.stderr.on('data', function(data) {
+    // we can't stop execution from this function
+    callback(new Error(data));
```

Comment:
A small shortcoming with this section is that the data event can fire more than once, causing the callback to be fired more than once.

We should use `childD.stderr.once` so that the event can't call the callback again.